src/routes/_components/sections/HeroSection.svelte

```diff
@@ -17,10 +17,21 @@
 	<FloatTicket
 		float={{
```

Comment:
It would be good to move this to config file where is super easy to change.
Is this a certificate on purpose?
Why don't we fetch a real FLOAT in this place?